zuul.d/ansible-cloud-jobs.yaml

```diff
@@ -335,10 +335,195 @@
     vars:
       ansible_collections_repo: "{{ zuul.project.canonical_name }}"
 
-
 ### Cloud Common
 - job:
     name: ansible-test-units-cloud-common-python38
     parent: ansible-test-units-base-python38
     vars:
       ansible_collections_repo: "{{ zuul.project.canonical_name }}"
+
+- job:
+    name: ansible-test-sanity-aws-ansible-2.9-python36
+    parent: unittests
+    nodeset: controller-python36
+    dependencies:
+      - name: build-ansible-collection
+        soft: true
+    pre-run: playbooks/ansible-test-base/pre.yaml
+    run: playbooks/ansible-test-base/run.yaml
+    required-projects:
+      - name: github.com/ansible/ansible
+        override-checkout: stable-2.9
+    timeout: 3600
+    vars:
+      ansible_test_collections: true
+      ansible_collections_repo: "{{ zuul.project.canonical_name }}"
+      ansible_test_command: sanity
+      ansible_test_docker: true
+      ansible_test_enable_ara: false
+      ansible_test_python: 3.6
+
+- job:
```

Comment:
Since basically, all these jobs are 99% the same, I would inherit them from the first in the list and just overwrite the two different keys.

Reply:
tbh I didn't entirely realize I could do that with `parent`!  Let's see if I did that right or not.  :)